include/svgpp/parser/grammar/color.hpp

```diff
@@ -71,6 +73,15 @@ class color_grammar: public qi::grammar<Iterator, typename ColorFactory::color_t
               >> *space 
               >> ')' 
             )
+        |   (
+                 detail::no_case_if_css(PropertySource())[ lit("rgba(") ]
+              >> *space
+              >> (  components_rgba_rule [_val = _1]
+                 |  percentage_rgba_rule [_val = _1]
+                 )
+              >> *space
+              >> ')'
+            )
```

Comment:
All changes must be under `#ifdef` (e.g. `#ifdef SVGPP_ACCEPT_RGBA_COLOR`) to not break backward compatibility. It makes sense to place changes together when it's reasonable.